sdks/python/build.gradle

```diff
@@ -197,13 +197,13 @@ task directRunnerIT(dependsOn: 'installGcpTest') {
 //    ./gradlew :beam-sdks-python:portableWordCount -PjobEndpoint=localhost:8099
 //
 task portableWordCount {
-  dependsOn portableWordCountTask('portableWordCountExample', project.hasProperty("streaming"))
+  dependsOn portableWordCountTask('portableWordCountExample', project.hasProperty("streaming"), project.hasProperty("crossLanguage"))
 }
 
-def portableWordCountTask(name, streaming) {
+def portableWordCountTask(name, streaming, crossLanguage) {
   tasks.create(name) {
     dependsOn = ['installGcpTest']
-    mustRunAfter = [':beam-runners-flink_2.11-job-server-container:docker', ':beam-sdks-python-container:docker']
+    mustRunAfter = [':beam-runners-flink_2.11-job-server-container:docker', ':beam-sdks-python-container:docker', ':beam-sdks-java-container:docker', ':beam-runners-core-construction-java:buildTestExpansionServiceJar']
```

Comment:
Should only require `':beam-sdks-java-container:docker', ':beam-runners-core-construction-java:buildTestExpansionServiceJar'` for the cross-language wordcount.

Reply:
For Flink it would also be better to run the Java expansion service as part of the job server and have the ability to discover expansion service address via the job service endpoint to cut down on the bits and pieces a user is exposed to.

Reply:
Agree, this is tracked here: https://jira.apache.org/jira/browse/BEAM-6731